src/sqlfluff/core/linter/linted_dir.py

```diff
@@ -57,19 +57,26 @@ def check_tuples(self, by_path: bool = False):  # pragma: no cover
         """Default overload method."""
         ...
 
-    def check_tuples(self, by_path=False):
+    def check_tuples(self, by_path=False, raise_on_non_linting_violations=True):
```

Comment:
Changed this test-related function so linter tests can include SQL files with parse errors.